poonam/JS/Assignment14_js/que2.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Bank details</title>
+    </head>
+    <body>
+        <script>
+            class BankDet {
+                constructor() {
+                    this.acc_no = 0;
+                    this.acc_name = "";
+                    this.bal = 0;
+                }
+                getData(acc_no, acc_name, bal) {
+                    this.acc_no = prompt("Enter Employee No.:");
+                    this.acc_name = prompt("Enter name:");
+                    this.bal = parseInt(prompt("Enter balance:"));
+                }
```

Comment:
You have to pass the value through parameter

Reply:
yes you have to intialize properties on the values passed.
getData(acc_no, acc_name, bal) {
                    this.acc_no =acc_no;
......
read the values with the help of prompt in the main flow of the program and pass it to the method/function